src/main/scala/org/broadinstitute/dsde/firecloud/model/ElasticSearch.scala

```diff
@@ -36,40 +45,44 @@ object Document {
 
 
 // classes to convert from json body and to json response
+
+/**
+  *
+  * @param searchString
+  * @param searchFields a map of field names to a list of possible values which will be part of the search criteria
+  * @param fieldAggregations the aggregation data to retrieve, results will differ based on the search criteria
+  * @param maxAggregations the default is 10, this should only be specified if the use has requested to see more options
+  * @param from used for pagination, where to start the returned results
+  * @param size used for pagination, how many results to return
+  */
```

Comment:
for Scala education (not requesting any PR changes): http://docs.scala-lang.org/style/scaladoc.html